care/facility/migrations/0468_patientnotes_root_note_alter_fileupload_file_type_and_more.py

```
@@ -0,0 +1,29 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('facility', '0467_alter_hospitaldoctors_area'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='patientnotes',
```

Ruff (0.8.0)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)